prqlc/prqlc-parser/src/lexer/mod.rs

```diff
@@ -428,195 +473,3 @@ fn end_expr() -> impl Parser<char, (), Error = Cheap<char>> {
 
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
 pub struct TokenVec(pub Vec<Token>);
-
-#[cfg(test)]
-mod test {
```

Comment:
(checked this and this moved to `test.rs`)

Reply:
👍